crates/typst-library/src/foundations/func.rs

```diff
@@ -42,47 +40,35 @@ use crate::foundations::{
 /// #list[A][B]
 /// ```
 ///
-/// Functions are a fundamental building block of Typst. Typst provides
-/// functions for a variety of typesetting tasks. Moreover, the markup you write
-/// is backed by functions and all styling happens through functions. This
-/// reference lists all available functions and how you can use them. Please
-/// also refer to the documentation about [set]($styling/#set-rules) and
-/// [show]($styling/#show-rules) rules to learn about additional ways you can
-/// work with functions in Typst.
+/// 関数はTypstにおいて基礎となる構成要素です。
+/// Typstは様々な組版タスクに応じた関数を提供しています。
+/// さらには、作成されるマークアップの裏側では関数が用いられており、すべてのスタイル設定は関数を介して行われます。
+/// このリファレンスでは利用可能なすべての関数とその使い方を示します。
+/// Typstで関数をさらに活用する方法については、[set]($styling/#set-rules)ルールおよび[show]($styling/#show-rules)ルールのドキュメントも参照して下さい。
```

Comment:
原文ではsetとshowに対してリンクをつけられていますが、和訳した際に「ルール」という文字を両方につけているため、「ルール」まで含めてリンクをつけてはいかがでしょうか？すぐ下の記述でもそうなっており、「および」というプレーン文字だけをリンク文字で挟む形になることで見た目もよくなると思います。
```suggestion
/// Typstで関数をさらに活用する方法については、[setルール]($styling/#set-rules)および[showルール]($styling/#show-rules)のドキュメントも参照して下さい。
```